book/index.html

```diff
@@ -78,7 +78,15 @@
         <div id="page-wrapper" class="page-wrapper">
 
             <div class="page">
-                
+                <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/katex.min.css" integrity="sha384-D+9gmBxUQogRLqvARvNLmA9hS2x//eK1FhVb9PiU86gmcrBrJAQT8okdJ4LMp2uv" crossorigin="anonymous">
```

Comment:
is the link generated automatically by mdbook?

Reply:
Everything in the `book/` folder is an automated build output, please don't review it.

The only actual change is introduce this file: `theme/header.hbs`

Reply:
The content of `theme/header.hbs` is the reccomended way to deploy KaTeX. The LaTeX/math renderer made by Khan Academy which is very very good.